dropwizard-metrics-influxdb/src/main/java/com/izettle/metrics/dw/PackageNameAbbreviator.java

```diff
@@ -0,0 +1,17 @@
+package com.izettle.metrics.dw;
+
+import java.util.regex.Pattern;
+
+public class PackageNameAbbreviator {
+
+    // Capture the first letter of all dot-separated words that comes before the first capitalized word. E.g:
+    // Input:      org.example.MyClass.myMethod
+    // Captures:   ^   ^
+    private Pattern pattern = Pattern.compile("([a-z])\\w+(?=.*\\.[A-Z])");
```

Comment:
Make it static. Matcher is not thread safe, Pattern is. 😄

Reply:
Fixed!